frontend/src/components/TransactionsList.tsx

```diff
@@ -53,6 +117,26 @@ function TransactionsList({
         />
       ) : (
         <>
+          {appId && transactions.length > 0 && (
+            <div className="mb-4 flex justify-end">
+              <DropdownMenu>
```

Comment:
This dropdown menu should be inline with the "Transactions" title so we don't add extra vertical space. Probably best to be moved to where the title is rendered